neural_compressor/torch/quantization/quantize.py

```diff
@@ -67,44 +66,3 @@ def quantize(
             logger.info(f"Start to apply {algo_name} on the model.")
             q_model = algo_func(q_model, configs_mapping, run_fn=run_fn, run_args=run_args)
     return q_model
-
-
-def quantize_dynamic(
```

Comment:
Once more thing, the `quantize_dynamic` was removed, but it seems like the CI didn't catch the import error.

https://github.com/intel/neural-compressor/blob/2145f7523244918e5b100fe70e281253e664dc6a/neural_compressor/torch/quantization/__init__.py#L15-L16

Reply:
Sure, I will remove the import, thx.